grafana/grafana
# logs-volume: remove custom timeout

fixes https://github.com/grafana/grafana/issues/44756

- currently for the log-volume functionality we use a custom timeout value. we do not want that, so it got removed, and now we rely on the "global" timeout value set by https://grafana.com/docs/grafana/latest/administration/configuration/#timeout
- we also normalized the error-message shown in the log-volume display, the error message will be created the same way as it is created for the log-panel itself, except, if it is too long, you get a show-details button that shows it (we can customize it later, for now i wanted a clean situation there)

how to test:
- we start a loki-database and the slow-proxy docker image
  - `make devenv sources=loki,slow_proxy ORIGIN_SERVER=http://host.docker.internal:3100 SLEEP_DURATION=7s`
- we set a timeout of 4seconds for grafana. add this to your `conf/custom.ini`
    ```
    [dataproxy]
    timeout = 4
    ````
- now setup a loki datasource that points to `http://localhost:3011` (where the slow-proxy listens)
- so, the slow-proxy will return results after 7seconds, grafana is set to only wait 4seconds
- go to explore-mode, choose slowproxy-loki-datasource, and run any query (for example `{level="info"}`)
- after 4 seconds, both the log-volume part and the log-part will timeout at the same time with an error message
- now modify the `conf/custom.ini` set timeout to `10`.
- so, the slow-proxy will return results after 7seconds, grafana is set to wait 10 seconds, things should work now
- repeat the loki query
- both the log-volume and the log part should finish successfully
